packages/core/src/app/ui/accordion/AccordionItem.tsx

```diff
@@ -46,6 +46,7 @@ const AccordionItem: FunctionComponent<AccordionItemProps> = ({
             className,
             { [classNameSelected]: isSelected }
         ) }
+        data-test={ `${itemId}_accordion-item` }
```

Comment:
Added a `data-test` in the accordion item for every accordion item however, what are people's thoughts

Reply:
Thanks for doing this. Could you flip it instead, e.g.: `accordion-item_${itemId}`

Reply:
updated